app/uk/gov/hmrc/helplinefrontend/config/AppConfig.scala

```diff
@@ -27,6 +27,7 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
 
   val welshLanguageSupportEnabled: Boolean = config.getOptional[Boolean]("features.welsh-language-support").getOrElse(true)
   val backCallEnabled: Boolean = config.getOptional[Boolean]("features.back-call-support").getOrElse(false)
+  val findMyNinoEnabled: Boolean = config.getOptional[Boolean]("features.find-my-nino.enabled").getOrElse(false)
```

Comment:
With the flag for PDV it was agreed with Chris and Paul that it wouldn't default to false ie it would be
config.get[Boolean]("features.find-my-nino.enabled")
then the flag set appropriately in application.conf, app-config-base and app-config-production.

Reply:
Done